notebooks/1_BayesianWorkflow/3_Ins_EndtoEndWorkflow.ipynb

```diff
@@ -0,0 +1,178 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Section 1.3 End to End Statistics workflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From Wikipedia  \n",
+    "_Statistics is a branch of mathematics dealing with data collection, organization, analysis, interpretation and presentation._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bayesian Versus ...\n",
+    "It should be noted that Bayesian Statistics is one methodology in the broad field of statistics/machine learning. Below we briefly summarize the differences between Bayesian Statistics and other commonly known Statistics/Machine Learning methodologies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Bayesian statistics versus frequentists\n",
+    "**Bayesians:** Given the data how how likely are my parameters?  \n",
+    "**Frequentists:** Given the parameters what is the probability of the observed data?\n",
+    "\n",
+    "#### Frequentist interpretation: How much of a planet is covered in water?\n",
+    "*TODO*: Figure out a way to twist the last problem to meet a frequentist framework"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Bayesian statistics versus \"Machine Learning\"\n",
+    "Machine learning can most broadly be thought as a set of algorithms which infer their \"optimal\" parameters from given data. Common examples include linear regressions, or neural networks. Bayesian Statistics can also be classified as a machine learning algorithm."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "| Activity          | Linear Regression                   |  Deep Learning (Classification) | Bayesian Statistics            |\n",
```

Comment:
I kind of like the comparison but at the same time it should be clear to the audience that you are comparing a framework (Bayesian statistics) with particular methods. In this sense Least Squares Linear Regression is conceptually _just a particular case_ of a Bayesian model (implemented using a shorcut).